frontend/src/components/place/PlaceCheckinModal.vue

```diff
@@ -93,27 +93,27 @@ export default {
 
         handleResponse: function(response, successMessage) {
             switch (response.status) {
-                case 201:
-                    this.$toast.open({
-                        type: 'is-success',
-                        message: successMessage
-                    });
-                    break;
-                case 400:
-                    this.$toast.open({
-                        type: 'is-danger',
-                        message: response.statusText
-                    });
-                    break;
-                default:
-                    this.$toast.open({
-                        type: 'is-danger',
-                        message: 'Something went wrong. Try again later'
-                    });
+            case 201:
```

Comment:
This is not ok for sure

Reply:
We have that code on `development branch` that I merged only.

Reply:
Fixed in Pool Request #227.